src/core/resolvers/idux.ts

```diff
@@ -33,6 +33,12 @@ export interface IduxResolverOptions {
    * @default 'default'
    */
   importStyleTheme?: string
+
+  /**
+   * The alias of '@idux` package.
+   * @default '@idux'
+   */
+   alias?: string
```

Comment:
```suggestion
   packageName?: string
```

Reply:
`${alias}/${packageName}/${dirname}`

Reply:
I see, how about `scope`

```suggestion
   scope?: string
```

Reply:
ok，to the scope

Reply:
@sxzz is ok